### PR TITLE
LPS-133426 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133426

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 1]
```diff
@@ -891,6 +891,49 @@ definition {
...
+			CustomFields.addCP(
+				customFieldName = "Text Field",
+				customFieldType = "Input Field",
+				customInputDataType = "Text",
+				resourceName = "Page");
+		}
+
+		task ("Assign the site role to new user") {
+			JSONRole.assignRoleToUser(
+				roleTitle = "Administrator",
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:912]: The Administrator has Custom Fields permission. Create a role with permissions to add pages and contents.

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update the case according to comments?

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update the case according to comments?

Hi @Tim-Cao, PR Updated. I've been wondering, is it necessary add a page via UI?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 2]
```diff
@@ -891,6 +891,60 @@ definition {
...
+		}
+
+		task ("Define the permissions of site role") {
+			for (var resourceActionIndex : list "ADD_LAYOUT,UPDATE") {
+				Permissions.definePermissionViaJSONAPI(
+					resourceAction = "${resourceActionIndex}",
+					resourceName = "com.liferay.portal.kernel.model.Layout",
+					roleTitle = "Roles Siterole Name",
+					roleType = "site");
+			}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:917]: Missing Access in Site and Asset Library Administration in Pages permission

**Comment by Tim-Cao** [comment]
Hi @ozysouza, See changes https://github.com/liferay-echo/liferay-portal/compare/03d40faff2518f290bfed00cf1960b1c14a058d3...b70c10b8b36f5dcdff61254dae3acd0f61b297a7

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**